src/cli/cli.cpp

```diff
@@ -2722,6 +2722,18 @@ template <> otError Interpreter::Process<Cmd("log")>(Arg aArgs[])
 {
     otError error = OT_ERROR_NONE;
 
+    /**
+     * @cli log level
+     * @code
+     * log level
+     * 1
+     * Done
+     * @endcode
+     * @par
+     * Get the log level.
+     * @par api_copy
+     * #otLoggingGetLevel
```

Comment:
The api_copy content for this one doesn't work well with the CLI page, and duplicates with "Get the log level."  You would need to either change the API description to not refer to a function and drop the custom description (preferred), or switch the api_copy to a `@sa` for the API and customize the description here only.